packages/docs/docs/docs/proxies.md

```diff
@@ -0,0 +1,68 @@
+---
+id: proxies
+title: Proxy Pattern
+---
+
+This article describes the "unstructured storage" proxy pattern, the fundamental building block of ZeppelinOS's contract upgradeability feature.
+
+Note: For a more in depth read, please see [blog.zeppelinos.org/proxy-patterns](https://blog.zeppelinos.org/proxy-patterns/), which discusses the need for proxies, goes into more technical detail on the subject, elaborates on other possible proxy patterns that were considered for zOS, and more.
+
+## Why upgradeability?
+
+By design, smart contracts are immutable. On the other hand, software quality heavily depends on the ability to upgrade and patch source code in order to make iterative releases. Even though blockchain based software profits signifficantly from the technology's immutability, still a certain degree of mutability is needed for bugfixing and potential product improvements. ZeppelinOS solves this apparent contradiction by providing an easy to use, simple and robust upgrade mechanism for smart contracts that can be controlled by any type of governance, be it a multi-sig wallet, a simple address or a complex DAO.
```

Comment:
We could mention here the word "opt-in". Maybe, "simple, robust and opt-in mechanism..."